MCprep_addon/conf.py

```diff
@@ -81,11 +75,14 @@ class MCprepEnv:
 	def __init__(self):
 		self.data = None
 		self.json_data: Optional[Dict] = None
-		self.json_path: Path = Path(os.path.dirname(__file__), "MCprep_resources", "mcprep_data.json")
-		self.json_path_update: Path = Path(os.path.dirname(__file__), "MCprep_resources", "mcprep_data_update.json")
+		self.json_path: Path = Path(
+			os.path.dirname(__file__), "MCprep_resources", "mcprep_data.json")
+		self.json_path_update: Path = Path(
+			os.path.dirname(__file__), "MCprep_resources", "mcprep_data_update.json")
 
 		self.dev_file: Path = Path(os.path.dirname(__file__), "mcprep_dev.txt")
-		self.languages_folder: Path = Path(os.path.dirname(__file__), "MCprep_resources", "Languages")
+		self.languages_folder: Path = Path(
+			os.path.dirname(__file__), "MCprep_resources", "Languages")
```

Comment:
I think we should have a `MCPREP_RESOURCES` constant just to reduce repetition. Then, we can just make a `Path` object for each of these like this:
```py
MCPREP_RESOURCES: Path =  Path(os.path.dirname(__file__), "MCprep_resources")
#...
self.languages_folder: Path = Path(MCPREP_RESOURCES, "Languages")
```

Not a blocker though

Reply:
Done